src/App.tsx

```diff
@@ -1,7 +1,32 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string | null,
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    key: null,
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', (e) => {
+      const keyUp = e.key;
+      const key = `The last pressed key is ${keyUp}`;
+
+      this.setState({ key });
+    });
+  }
+
+  render() {
+    const { key } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__message">
+          {key === null ? 'nothing was pressed yet' : key }
```

Comment:
```suggestion
          {key === null ? 'Nothing was pressed yet' : key }
```

Reply:
Please check the test results before sending the PR 
![image](https://user-images.githubusercontent.com/98764641/236752510-2f53de23-5c07-416a-b546-beb9a5ddbbf1.png)